float32[] distance
---
bool success
moveit_msgs/RobotTrajectory plan